.github/workflows/shared-build-and-test.yaml

```diff
@@ -5,6 +5,10 @@ on:
       java_version:
         type: string
         default: '11'
+      skip_tests:
```

Comment:
Can I suggest we rename this to something like only_do_vulnerability_scan. Skips tests implies a build step which is not being done

Reply:
Ah makes sense, let me update